### PR TITLE
Add Add source.user aspect to mixer manifest

### PR DESCRIPTION
**Pull request by wattli**
**Release note**:

```release-note
NONE
```

---
This change is [Reviewable](https://reviewable.io/reviews/istio/mixer/1223)

### CODE REVIEW

**Comment by wattli** [comment]
@douglas-reid @mandarjog , could you take a look please? Thanks.

**Comment by douglas-reid** [comment]
can you also add it to https://github.com/istio/mixer/blob/master/testdata/config/attributes.yaml (new form of the config) ?

**Comment by wattli** [comment]
@douglas-reid , attributes.yaml is updated as well. I built the new image locally and it works.

File: testdata/configroot/scopes/global/descriptors.yml [hunk 1]
```diff
@@ -16,6 +16,8 @@ manifests:
         valueType: STRING
       source.serviceAccount:
         valueType: STRING
+      source.user:
```
Comment (douglas-reid) [inline testdata/configroot/scopes/global/descriptors.yml]: i just noticed that this in the kubernetes manifest... 

let's move it to the istio-proxy section (here and in the other file).

Comment (wattli) [reply testdata/configroot/scopes/global/descriptors.yml]: Just followed other source.*. Done.

**Comment by douglas-reid** [comment]
@mandarjog in addition to the 1000 other things you are doing today, can you help get this PR in... it is needed for auth testing, etc.

**Comment by wattli** [comment]
Thank you both.

**Pull request merged**